### PR TITLE
[DS-299]: [Pyxis React, Pyxis Elm, Pyxis SCSS] Change Html in Select component

### PR DESCRIPTION
**Pull request by lauratrapani**
https://prima-assicurazioni-spa.myjetbrains.com/youtrack/issue/DS-299

## 🐛 Bug Fix
- `pyxis-react`: The Select dropdown now has the correct keyboard interaction. 
- `pyxis-react` and `pyxis-elm`: if the user interacts with the keyboard and after with the mouse, the interaction happens as expected.

## 🏠 Internal
- `pyxis-elm`: The Html now has the same ARIA attribute as the same component in React
- `pyxis-scss` : The scrollbar is now visible in the dropdown of `Select` and `Autocomplete` component. The `.form-field--with-select-dropdown` has been renamed to `.form-field--with-custom-dropdown`.

### CODE REVIEW

File: packages/pyxis-react/src/components/Form/Select/Select.tsx [hunk 1]
```diff
@@ -1,11 +1,11 @@
 import React, {
-  FC, SelectHTMLAttributes, useState, KeyboardEvent, useEffect,
+  FC, SelectHTMLAttributes, useState, KeyboardEvent, useEffect, useRef,
 } from 'react';
 import classNames from 'classnames';
 import { IconChevronDown } from 'components/Icon/Icons';
 import useClickOutside from 'commons/hooks/useClickOutside';
 import useIsUnderBreakpoint from 'commons/hooks/useIsUnderBreakpoint';
-import Dropdown, { Item } from './Dropdown/Dropdown';
+import Dropdown, { Item } from 'internalComponents/Dropdown/Dropdown';
```
Comment (emanuelemacri) [inline packages/pyxis-react/src/components/Form/Select/Select.tsx:8]:
```suggestion
import Dropdown, { Item } from 'internalComponents/Dropdown';
```

**Review by emanuelemacri — CHANGES_REQUESTED** [review]
one little thing

File: packages/pyxis-elm/src/Pyxis/Components/Field/Select.elm [hunk 2]
```diff
@@ -133,6 +133,7 @@ type Msg
     | OnFocus
     | OnKeyDown KeyDown.Event
     | OnSelect String
+    | OnMouseEnter Option
```
Comment (alessiosantangelo) [inline packages/pyxis-elm/src/Pyxis/Components/Field/Select.elm:136]: Metterei in ordine alfabetico

File: packages/pyxis-elm/src/Pyxis/Components/Field/Select.elm [hunk 3]
```diff
@@ -855,3 +884,24 @@ pickOptionValue (Option { value }) =
...
+-}
+initializeActiveOption : Model msg -> Model msg
+initializeActiveOption ((Model { selectedValue, options }) as model) =
+    case selectedValue of
+        Just currentValue ->
+            options
+                |> List.Extra.find (getOptionData >> .value >> (==) currentValue)
+                |> PrimaFunction.flip setActiveOption model
+
+        _ ->
```
Comment (alessiosantangelo) [inline packages/pyxis-elm/src/Pyxis/Components/Field/Select.elm:899]: Meglio usare `Nothing`qui

File: packages/pyxis-react/src/stories/components/Select/Select.stories.tsx [hunk 4]
```diff
@@ -37,6 +37,22 @@ const items = [
     value: "option-4",
     label: "Option 4"
   },
+  {
```
Comment (alessiosantangelo) [inline packages/pyxis-react/src/stories/components/Select/Select.stories.tsx:40]: Magari questo array autogeneriamolo invece che scriverlo a mano

**Review by alessiosantangelo — CHANGES_REQUESTED** [review]
Due cavolatine, il resto LGTM

**Review by Gsciarra — APPROVED** [review]

**Review by alessiosantangelo — DISMISSED** [review]

**Review by emanuelemacri — APPROVED** [review]

**Review by emanuelemacri — APPROVED** [review]

**Pull request merged**